services/blockchain-connector/package.json

```diff
@@ -35,6 +35,8 @@
     "@liskhq/lisk-cryptography": "4.0.0-alpha.3",
     "big-json": "^3.1.0",
     "bluebird": "^3.7.2",
+    "decompress": "^4.2.1",
+    "decompress-targz": "^4.1.1",
```

Comment:
Please try to get `tar` dependency working else let's figure out something else. These dependencies have not been maintained for a very long time.

Reply:
resolved with `tar`. Thanks a lot @sameersubudhi for the 🪄